### PR TITLE
Add `addModulesDirectories` config

### PR DESCRIPTION
**Pull request by neezer**
Allows appending to the default list of modules directories. I picked a name that I think sounds good, but happy to change it if y'all have something else you'd prefer. 😄 

Relevant discussion: https://github.com/postcss/postcss-import/issues/195

@RyanZim

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
Maybe we could refactor `options.resolve` for that , instead of adding it to options directly?

```js
let resolveOpt = {} // defaults

if (typeof options.resolve === 'object') {
   resolveOpt = Object.assign(resolveOpt, options.resolve)
   resolveOpt.moduleDirectories.push('node_modules', 'web_modules')
}

if (typeof options.resolve === 'function') {
  // Use custom resolver
}
```

**Comment by neezer** [comment]
Neat, @michael-ciniawsky! Happy to change the PR to this approach.

@RyanZim @MoOx Any preference?

**Comment by RyanZim** [comment]
@michael-ciniawsky I like that idea, I just think using the `resolve` option is rather implicit behavior.

If we go that direction, we should set `root`, and `path` in the same object as `moduleDirectories`.

For right now, I would be in favor of just doing it the way it is done in this PR. For the next major version, I would consider removing `root`, `path`, and `addModuleDirectories`, in favor of a single `resolveOpts` option; bikeshed about that later.

File: README.md [hunk 1]
```diff
@@ -198,6 +198,19 @@ postcss: function(webpack) {
 }
 ```
 
+#### `addModulesDirectories`
+
+Type: `Array`  
+Default: `[]`
+
+An array of folder names to add to [Node's resolver](https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders).
```
Comment (RyanZim) [inline README.md]: Probably should link to https://github.com/substack/node-resolve instead, since that is the module that is actually used.

Comment (neezer) [inline README.md]: @RyanZim Updated!

**Comment by RyanZim** [comment]
@ai Thoughts? @MoOx Any objections?

**Comment by RyanZim** [comment]
@neezer Thanks!

**Comment by renatorib** [comment]
It's not published yet?

**Comment by RyanZim** [comment]
@renatorib It's not. I should publish it. I was planning on adding a few other things to the next release, but I haven't got around to it, so I guess I'll just publish this. I'll comment here as soon as I get it done.

**Comment by RyanZim** [comment]
@renatorib @neezer Released: https://github.com/postcss/postcss-import/releases/tag/9.1.0. Sorry for the long wait.

**Comment by neezer** [comment]
@RyanZim Great! So should #195 be closed now?

**Comment by RyanZim** [comment]
@neezer Yep, thanks.

**Pull request merged**